schema.sql

```diff
@@ -7,4 +7,7 @@ CREATE TABLE animals(
     escape_attempts INT NOT NULL,
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL
-);
```

Comment:
Although I am highlighting this file, this change should be done to the _**README.md**_ file. 

- [ ] Kindly fix the link to the LICENSE because anytime I click on it, it displays a 404 error page. Having a valid LICENSE helps people know if they have legal rights to use your project or codes in it. 👍 😎

Reply:
Implemented